src/main/java/Duke.java

```diff
@@ -1,10 +1,274 @@
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+
 public class Duke {
+    static List<Task> tasks = new ArrayList<>();
+    static int numTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        initialiseTasks();
+
+        System.out.println("Hello! I'm KaiKai.");
+        System.out.println("What can I do for you?");
+        System.out.println("______________________________________");
+
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+
+
+        while (!str.equals("bye")) {
+            switch (str) {
+            case "list":
+                printTasks();
+                break;
+            case "mark":
+                markTask();
+                break;
+            case "unmark":
+                unmarkTask();
+                break;
+            case "add":
+                System.out.println("Okie, what type of task is this?");
+                System.out.println("1.Todo");
+                System.out.println("2.Deadline");
+                System.out.println("3.Event");
+                str = sc.nextLine();
+                switch (str) {
+                case "1":
```

Comment:
Avoid magic numbers. Consider putting the integer into a variable (e.g. `TASK_TYPE_INDEX`)